CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -774,6 +774,47 @@ function PlayerHospital:playSound(sound)
   end
 end
 
+--! The UI parts of earthquake ticks
+--!param stage (string) Stage of the active earthquake
+function PlayerHospital:tickEarthquake(stage)
```

Comment:
The trouble here is an earthquake is a world event. The announcements aren't though.

Reply:
These are the UI parts of the world event, though there may be more from competitor hospitals later.